cmake/scripts/darwin_embedded/ArchSetup.cmake

```diff
@@ -47,6 +47,10 @@ set(CMAKE_XCODE_ATTRIBUTE_INLINES_ARE_PRIVATE_EXTERN OFF)
 set(CMAKE_XCODE_ATTRIBUTE_GCC_SYMBOLS_PRIVATE_EXTERN OFF)
 set(CMAKE_XCODE_ATTRIBUTE_COPY_PHASE_STRIP OFF)
 
+if(NOT TARBALL_DIR)
+  set(TARBALL_DIR "/Users/Shared/xbmc-depends/xbmc-tarballs")
```

Comment:
Not sure if it's a good idea to default to outside of the project directory.

Reply:
This is already the case on apple platforms for tools/depends usage. This is just parity with that existing use case in a cmake world. I dont think its worth changing how that works as it is right now.

Reply:
That's only the case for the jenkins build steps.

https://github.com/xbmc/xbmc/blob/387df8469f712712bb0865d8095380874c3c985e/tools/buildsteps/tvos/configure-depends#L8

Otherwise when running the configure command you are free to set it wherever with the `--with-tarballs` switch.

Reply:
https://github.com/xbmc/xbmc/blob/387df8469f712712bb0865d8095380874c3c985e/tools/depends/configure.ac#L192-L194

Reply:
Just to note, these decisions predate my involvement. however default location for apple has prefix set, which then obviously sets tarballs location when not overridden explicitly.